src/objects.rs

```diff
@@ -775,32 +775,14 @@ pub enum ExceptionType {
     Remove,
 }
 
-pub type CalendarDates = Vec<(Date, ExceptionType)>;
+use std::collections::BTreeSet;
 
 #[derive(Serialize, Deserialize, Debug, PartialEq)]
 pub struct Calendar {
-    #[serde(rename = "service_id")]
+    #[serde(skip)]
```

Comment:
please remove these skip. It's useless for gtfs/nffs, and may be useful someday if we want to support serde on the ntm::Model.